web/pages/api/v0/_types.ts

```diff
@@ -140,3 +141,61 @@ function augmentAnswerWithProbability(
     probability,
   }
 }
+
+export type LiteUser = {
+  id: string
+  createdTime: number
+
+  name: string
+  username: string
+  url: string
+  avatarUrl?: string
+
+  bio?: string
+  bannerUrl?: string
+  website?: string
+  twitterHandle?: string
+  discordHandle?: string
+
+  balance: number
+  totalDeposits: number
+  totalPnLCached: number
```

Comment:
I'm not especially sure how long we want to commit to supporting all these values; `creatorVolumeCached` and `totalPnLCached`, but also `totalDeposits` and maybe even `balance`.

The reason is that these are all denormalized values, basically cached here for convenience; they should be fully rederivable from txn history. If API clients start depending on these, we still want the flexibility to change this in the future.

But I think this is fine to merge as-is, we might just want to make a note in the doc